### DISCUSSION TITLE
I want to add Multicenter and variable attachment to a molecule

### DISCUSSION THREAD

**Discussion by shyamramesh** [rdkit/rdkit#6760] [Q&A]
Can someone help me with how to do this in python
I want to add Multi-center and variable attachment points to a molecule?

**Comment by greglandrum** [comment]
It's not particularly trivial to do this manually. The best approach is to start with a v3000 mol file which has the multi-center attachments in it already. I guess you don't have one of those?

**Reply by shyamramesh** [reply]
Thankyou. Can you point me to a python sample, if possible.

**Reply by bertiewooster** [reply]
These two blog posts have Python code to create multiple attachment points: https://greglandrum.github.io/rdkit-blog/posts/2021-05-13-intro-to-the-molecule-enumerator.html and https://greglandrum.github.io/rdkit-blog/posts/2025-02-08-more-on-generalized-substructure-search.html.
